src/test_l1_dsHdmiIn.c

```diff
@@ -2079,22 +2203,30 @@ void test_l1_dsHdmiIn_negative_dsGetAVLatency(void) {
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
     // Step 1: Call dsGetAVLatency() without initializing the HDMI input sub-system
     int audioLatency, videoLatency;
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     UT_ASSERT_EQUAL(dsGetAVLatency(&audioLatency, &videoLatency), dsERR_NOT_INITIALIZED);
+    #endif
 
     // Step 2: Initialize the HDMI input sub-system using dsHdmiInInit()
     UT_ASSERT_EQUAL(dsHdmiInInit(), dsERR_NONE);
 
     // Step 3: Call dsGetAVLatency() with valid inputs (NULL, int*)
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     DS_ASSERT_AUTO_TERM_NUMERICAL(dsGetAVLatency(NULL, &videoLatency), dsERR_INVALID_PARAM);
+    #endif
 
     // Step 4: Call dsGetAVLatency() with valid inputs (int*, NULL)
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     DS_ASSERT_AUTO_TERM_NUMERICAL(dsGetAVLatency(&audioLatency, NULL), dsERR_INVALID_PARAM);
```

Comment:
In the else case, we need to handle the expected dsERR_NONE

Reply:
done